frontend/test/__support__/testDataset.ts

```diff
@@ -1,19 +1,21 @@
 import type { DatasetColumn, DatasetData } from "metabase-types/api";
 import { createMockDatasetData } from "metabase-types/api/mocks";
+import { PRODUCTS_ID, PRODUCTS } from "metabase-types/api/mocks/presets";
 
 const testColumns: DatasetColumn[] = [
   {
     description:
       "The numerical product number. Only used internally. All external communication should use the title or EAN.",
     semantic_type: "type/PK",
-    table_id: 1,
+    table_id: PRODUCTS_ID,
```

Comment:
avoid using hard-coded values and use our defined constants instead since it's more standardized.